src/Portal/Portal/Program.cs

```diff
@@ -30,13 +42,6 @@ public static void Main(string[] args)
 
                     builder.AddAzureAppConfiguration(azureAppConfConnectionString)
```

Comment:
```suggestion
                     builder.AddAzureAppConfiguration(options =>
                        {
                            options.Connect(azureAppConfConnectionString)
                                .ConfigureRefresh(refreshOptions =>
                                {
                                    // Only refresh the logging level, so set refreshAll to false
                                    refreshOptions.Register("logging:level", false);
                                    // Set a timeout for the cache so that it will poll the Azure config every X timespan.
                                    refreshOptions.SetCacheExpiration(cacheExpirationTime: new System.TimeSpan(0, 0, 0, 15, 0));
                                }); ;
                        })
```
This will allow places using IOptionsSnapshot to update log level live. 

Would also suggest moving from a key section `logging` to filters specifically for each application.

Reply:
To clarify on last point, I don't think we want all applications sharing the same log level.